src/view-data/method.ts

```diff
@@ -113,9 +113,15 @@ function getPathToMethodName(httpVerb: string, path: string): string {
   });
 
   const result = camelCase(segments.join("-"));
-  return `${httpVerb.toLowerCase()}${result[0].toUpperCase()}${result.substring(
-    1
-  )}`;
+  const lowerCasedHttpVerb = httpVerb.toLowerCase();
+  if (result.length > 0) {
+    return `${lowerCasedHttpVerb}${result[0].toUpperCase()}${result.substring(
+      1
+    )}`;
```

Comment:
lets put the "result[0].toUpperCase()}${result.substring(1)" part in a helper const as well for better readability